config/v1.5/aws-k8s-cni-cn.yaml

```diff
@@ -77,6 +77,7 @@ spec:
                     operator: In
                     values:
                       - amd64
+                      - arm64
```

Comment:
Why are you updating to `arm64` in this file? Unless `961992271922.dkr.ecr.cn-northwest-1.amazonaws.com.cn/amazon-k8s-cni:v1.5.7` is actually multi-arch we shouldn't be doing this.